02_activities/assignments/Assignment2_rubric.md

```diff
@@ -0,0 +1,70 @@
+# Assignment 2: Design a Logical Model and Advanced SQL
+
+## General
+  - Weight: *70 points*
+  - Criteria: Participant Name on Assignment
+  - Criteria: Assignment is free of noticeable typos
+  - Criteria: Written content for Section 1 is no longer than two pages
+  - Criteria: Two ERDs exist
+  - Criteria: assignment2.sql is completed
+  - Criteria: assignment2.sql is correct
+
+## Section 1
+- *(50 points total)*
```

Comment:
When the Markdown gets rendered, since the sections aren't visually defined, the whole 50 points here, 20 points there stuff gets confusing. Since you're already saying at the top 70 points total, don't put the section points, just put the leaf points. Meaning the points for every single specific section, makes it less confusing, and therefore we'll get fewer questions asking to clarify.

Reply:
from my chat with Dan:

its a good comment -- 
the rubric with the sub-bullet points are actually for the LS
the participants dont need this level of detail
so i'll push the current version to the instructor section for LS, and remove for participants